enhancements/windows-containers/cluster-logging-windows-worker-nodes.md

```diff
@@ -36,116 +35,87 @@ and pods running in an OpenShift cluster
-    containers to redirect logs to standard out from various sources like 
-    ETW, Perf Counters etc
-* Customization of log collection infrastructure
-  * We plan to support fluentd only
-  * Microsoft will provide us with a Windows fluentd binary or gem file
+* Logging for Windows containers.
+* Customization of log collection infrastructure.
+  * We plan to use the Microsoft provided Windows Fluentd binary and the Fluentd
+    logging driver that enables Windows containers to redirect logs to various
+    destinations using the Fluentd output plugins.
 
 
 ## Proposal
 
-The main idea here is to deploy fluentd as a log collection agent onto Windows 
-nodes after cluster logging operator is deployed onto the cluster. Fluentd has
-support for Windows logging. We plan to leverage it and run fluentd as a 
-Windows service.
+The main idea here is to deploy Fluentd as a log collection agent onto Windows 
+nodes after cluster logging operator is deployed onto the OpenShift cluster. 
+Fluentd has support for Windows logging. We plan to leverage it and run Fluentd 
+as a Windows service.
 
 
 ### Implementation Details
 
-We plan to create a Windows Node logging Ansible Playbook that can
-
-* Download fluentd binary for Windows from a known location and transfers it to
-  the Windows node
-* Configure the container runtime to use fluentd as the logging driver
-* Configure fluentd to communicate with elasticsearch data store in the same 
-  format as logs emitted from linux worker nodes
-
-The inputs to this playbook
-
-* The ip address, port, certificate to connect the elasticsearch service
-* URL from which fluentd Windows binary can be downloaded from
-
-We can get the ipaddress and other details from various pods and secrets present
-in the OpenShift cluster, once the cluster logging operator has been deployed.
-If the certificates change, cluster admin should run the ansible playbook
-again. 
-
-Kubelet on Windows nodes logs to a file. So, we also need to configure 
-fluentd's record transformer to ensure that logs collected from the kubelet log
-file are in format understandable to elasticsearch
+Once WMCO is installed from the OperatorHub and Cluster Logging operator is deployed 
+onto the cluster, the WMCO will perform the following steps to configure logging:
+
+ * Download the td-agent .msi installer binary from a known location and 
+      transfer it to the Windows node.
+ * Run td-agent's fluentdwinsvc as a Windows service.
+         * td-agent is a stable distribution of Fluentd, fluentdwinsvc
+           is permanently registered as a Windows service by the msi installer.
+ * Install the required Fluentd output plugins onto the Windows nodes.
+ * Configure the container runtime to use Fluentd as the logging driver on Windows node.
+ * Configure td-agent to forward logs to the elasticsearch data storage in a format
+   understandable to elasticsearch. 
+ * Kubelet on Windows nodes logs to a file. So, we also need to configure 
```

Comment:
How does this differ from Linux? (question for me, not to be written in the enhancement)